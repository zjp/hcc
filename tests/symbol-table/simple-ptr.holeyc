boolptr glob;
int main(){
	int a;
	glob = a;
}
