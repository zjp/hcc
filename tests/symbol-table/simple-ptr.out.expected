boolptr glob;
int main(){
	int a;
	glob(boolptr) = a(int);
}
